src/cysignals/implementation.c

```diff
@@ -312,10 +323,13 @@ static void setup_trampoline(void)
     size_t trampolinestacksize = 1 << 16;
 
     while (trampolinestacksize < PTHREAD_STACK_MIN) trampolinestacksize *= 2;
+#ifdef POSIX
```

Comment:
Replace this by a configure check for `mmap`

Reply:
Why ? If not on POSIX `sys/mman.h` is not included. 

Do you want to create a macro like `HAS_MMAP` or use an existing one ( `#ifdef MAP_ANONYMOUS`) ?